docs/swagger-spec/api.yml

```diff
@@ -200,8 +200,7 @@ definitions:
         type: object
         description: key value map with connection details (Service manager won't interpret the details of the map it will send this back to the requester as is)
         additionalProperties:
-          items:
-            type: string
+          type: object
```

Comment:
I am hesitant to change this. The original form was what crashed swagger. Using type `string` would change the map-type in the relevant generated structs from `map[string]interface{}` to `map[string]string` which is wrong for our code. As is, with type `object` we get the proper type for the go structures.

We are using the openapi which is vendored in the submoduled swagger. All the GOPATH shenanigans are in support of that, because only then do we know that swagger and openapi match.

Reply:
Ah, if that makes things harder, I guess we can do it later… needs a card to investigate though 😢